tests/tensor_tests.rs

```diff
@@ -132,13 +132,25 @@ fn cat_and_stack() {
 
 #[test]
 fn onehot() {
```

Comment:
Could we keep the original test as it was already helpful, and maybe add a new test that is protected by `has_mps` (as I don't know of any guarantees when it comes to using the mps device when `has_mps` returns false)?

Reply:
I dropped the `Mps` test as I don't have an Mps machine atm to confirm the test. I modified the test to show that it at least applies to both int and float tensors.

Reply:
done.